From 9b5f228f5ebd20eed794c527d2e37fcaded8bd09 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20P=C3=A1ral?= <kamil.paral@gmail.com>
Date: Mon, 25 Aug 2025 22:00:47 +0200
Subject: [PATCH] 70-mouse.hwdb: Add Razer Basilisk V3, Asus Cerberus, +2 more

All mice were measured using mouse-dpi-tool, and the measurements match vendors
specs, with the exception of Asus Cerberus (it officially has
500/*1000/1500/2500 DPI, but my measurements were quite different, so I opted
to include the real values).

(cherry picked from commit 2f615ec1d1a597586af15e6e94c0b3503a7da108)

Resolves: RHEL-72702
---
 hwdb.d/70-mouse.hwdb | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index da64f4837e..dc9be4f915 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -191,6 +191,14 @@ mouse:bluetooth:v256fpc63a:name:*
 mouse:bluetooth:v05acp030d:name:*:*
  MOUSE_DPI=1300@1000
 
+##########################################
+# Asus
+##########################################
+
+# Asus Cerberus
+mouse:usb:v04d9pa0d6:name:E-Signal/A-One USB Gaming Mouse:*
+ MOUSE_DPI=800@500 *1350@500 1600@500 2650@500
+
 ##########################################
 # Cherry
 ##########################################
@@ -388,6 +396,10 @@ mouse:usb:v17efp6045:name:Lenovo USB Laser Mouse:*
 mouse:usb:v17efp6044:name:ThinkPad USB Laser Mouse:*
  MOUSE_DPI=1200@125
 
+# Lenovo Essential USB Mouse (SM-8823)
+mouse:usb:v17efp608d:name:PixArt Lenovo USB Optical Mouse:*
+ MOUSE_DPI=1600@125
+
 ##########################################
 # Logitech
 ##########################################
@@ -675,6 +687,10 @@ mouse:usb:v046dpc517:name:Logitech USB Receiver:*
 mouse:usb:v046dpc046:name:Logitech USB Optical Mouse:*
  MOUSE_DPI=1000@125
 
+# Logitech RX1500
+mouse:usb:v046dpc061:name:Logitech USB Laser Mouse:*
+ MOUSE_DPI=1000@125
+
 # Logitech M100 Optical Mouse
 mouse:usb:v046dpc05a:name:Logitech USB Optical Mouse:*
  MOUSE_DPI=1000@125
@@ -837,6 +853,10 @@ mouse:usb:v1532p0029:name:Razer Razer DeathAdder:*
 mouse:usb:v1532p00a3:name:Razer Razer Cobra:*
  MOUSE_DPI=400@1000 800@1000 *1600@1000 3200@1000 6400@1000
 
+# Razer Basilisk V3
+mouse:usb:v1532p0099:name:Razer Razer Basilisk V3:*
+ MOUSE_DPI=400@125 800@125 1600@125 3200@125 6400@125 400@500 800@500 1600@500 3200@500 6400@500 400@1000 800@1000 *1600@1000 3200@1000 6400@1000
+
 ##########################################
 # Roccat
 ##########################################
